packages/samples/headless-react/src/components/date-facet/date-facet.class.tsx

```diff
@@ -51,7 +51,11 @@ export class DateFacet extends Component<DateFacetProps, DateFacetState> {
       return null;
     }
 
-    if (!this.state.values.length) {
+    if (
```

Comment:
Range facets have values with 0 results, we have to check if there are selected or values with results

Reply:
I think we should extract this into a separate function and possibly add a comment explaining this.